index.html

```diff
@@ -1392,10 +1379,10 @@ <h3>Identifiers</h3>
         <dl>
           <dt><dfn class="export">id</dfn></dt>
           <dd>
-The value of the `id` [=property=] MUST be a single [=URL=].
-It is RECOMMENDED that the [=URL=] in the `id` be one which, if
-dereferenced, results in a document containing machine-readable information
-about the `id`.
+The `id` [=property=] is OPTIONAL. If present, the value of the `id`
+[=property=] MUST be a single [=URL=] which MAY be dereferenced. It is
```

Comment:
This contradicts the above statement that the `id` value can be a GUID.

Reply:
How does it contradict: "urn:uuid:50e8098a-029a-11ef-a8e7-071a217fc2a9"?

Reply:
Its not dereferenceable

Reply:
The text says "If present, the value of the `id` [=property=] MUST be a single [=URL=] which MAY be dereferenced."

We could say:

```suggestion
[=property=] MUST be a single [=URL=], which MAY be dereferenced. It is
```

OR

```suggestion
[=property=] MUST be a single [=URL=], which MAY be dereferenceable. It is
```

Which one would you prefer? Or would you prefer alternate text. If you would prefer alternate text, please suggest something, that will make the chance process go more rapidly.

Reply:
I think this is just a typo that should read `MAY be dereferenceable` (just as the following sentence says).

```suggestion
[=property=] MUST be a single [=URL=] which MAY be dereferenceable. It is
```